src/cli/print/print.go

```diff
@@ -6,10 +6,12 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/fatih/color"
```

Comment:
could we wrap the `fatih/color` interface in `internal/common/color`?  Would be nicer to run everything through a single package's interface.  Will also make swapping out the package easier to do, if we ever need to go that routet.